rules-tests/DeadCode/Rector/Assign/RemoveUnusedVariableAssignRector/Fixture/foo.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Rector\DeadCode\Tests\Rector\Assign\RemoveUnusedVariableAssignRector\Fixture;
+
+class Foo {
+    public function bar(string $text, bool $flag): string {
+        $text = str_replace('a', 'b', $text);
+        $text = str_replace('c', 'd', $text);
+        if ($flag) {
+            $text = str_replace('d', 'e', $text);
+        }
+        else {
+            $text = str_replace('d', 'f', $text);
+        }
```

Comment:
The fixture should be as small as possible, to ease the fix writing.

How is this part relevant to the test case? Could it be removed without making test pass?